frame/staking/src/pallet/mod.rs

```diff
@@ -1773,6 +1789,36 @@ pub mod pallet {
 			MinCommission::<T>::put(new);
 			Ok(())
 		}
+
+		/// Migrates an account's `RewardDestination` in `Payee` to `PayoutDestination` in `Payees`
+		/// if a record exists and if it has not already been migrated.
+		///
+		/// Effects will be felt instantly (as soon as this function is completed successfully).
+		///
+		/// The dispatch origin for this call can be signed by any account.
+		///
+		/// ## Complexity
+		/// - O(1)
+		/// - Independent of the arguments. Insignificant complexity.
+		/// - Contains a limited number of reads.
+		/// - Writes are limited to the `who` account key.
+		#[pallet::call_index(26)]
+		#[pallet::weight(T::WeightInfo::update_payee())]
+		pub fn update_payee(origin: OriginFor<T>, controller: T::AccountId) -> DispatchResult {
```

Comment:
This will need to be called 69k times? Might be significant fees overall? 
I believe we can do this migration on `on_idle` as well and guarantee that migration will finish (until https://github.com/paritytech/substrate/pull/14275 is ready ❤️ ).  Also once migration is done, this call is not useful so probably better to try doing it `on_idle`.

Reply:
This call only updates one record at a time. My plan is to just run a bot and do the batching on the front end. I understand we have batched on the pallet side before, like with a recent account migration, but to keep things simple and low weight I just opted for a single update per call.

Reply:
> until https://github.com/paritytech/substrate/pull/14275 is ready ❤️

Indeed, looking forward to see this in action!

Reply:
Another option is to open an OpenGov proposal to update all the records via `update_payee`.